pkg/cmd/run/list/list.go

```diff
@@ -148,7 +158,7 @@ func listRun(opts *ListOptions) error {
 
 		tp.AddField(run.CommitMsg(), nil, cs.Bold)
 
-		tp.AddField(run.Name, nil, nil)
+		tp.AddField(IDToName[run.WorkflowID], nil, nil)
```

Comment:
Since `run.Name` doesn't seem safe to use, should we remove the `run.Name` field altogether and expose `run.WorkflowName()` that resolves the correct workflow name?

Reply:
I changed it so that we're populating the run.Name with the correct name field, let me know what you think